app/src/main/java/com/owncloud/android/utils/SyncedFolderUtils.kt

```diff
@@ -129,7 +130,8 @@ object SyncedFolderUtils {
         return when {
             fileName != null -> {
                 !DISQUALIFIED_MEDIA_DETECTION_FILE_SET.contains(fileName.lowercase()) &&
-                    !fileName.startsWith(THUMBNAIL_DATA_FILE_PREFIX)
+                    !fileName.startsWith(THUMBNAIL_DATA_FILE_PREFIX) &&
+                    !fileName.startsWith(EXTERNAL_TRASHCAN_PREFIX)
```

Comment:
```suggestion
                    !fileName.startsWith(EXTERNAL_TRASHCAN_PREFIX) &&
                    !fileName.startsWith(EXTERNAL_PENDING_PREFIX)
```
Can you add `.pending` file exclusion too